frontend/device/kobo/device.lua

```diff
@@ -233,7 +233,12 @@ function Kobo:resume()
 end
 
 function Kobo:powerOff()
-    os.execute("poweroff")
+    local lfs = require("libs/libkoreader-lfs")
+    if lfs.attributes("/sbin/poweroff", "mode") == "file" then
+        os.execute("/sbin/poweroff")
```

Comment:
Is there any case that `/sbin` not in the `PATH` env variable such that `os.execute("poweroff")` is not sufficient?